Automattic/wp-calypso
# A4A > Agency Tier: Implement download tier badges

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1296

## Proposed Changes

This PR implements download tier badges functionality. 

NOTE: we might have to update the logo

## Why are these changes being made?

* To allow agencies to download their tier badges for all the directories (WordPress.com, WooCommerce, Jetpack & Pressable)

## Testing Instructions

* Open the A4A live link
* Add all four directories to the Partner Directory (/partner-directory/agency-expertise). You don't have to accept it. We are currently not filtering it. This will be done in another PR to make the testing easy. 
* Update your agency tier to **Emerging Partner** using the MC tool > Go to the Agency Tier page & verify the download badges button is not visible on the top right corner of the page.

* Change the agency tier to **Agency Partner** > Refresh the UI and verify the screen looks as shown below. Click the `Download your badges` button > Verify all the directory badges are visible for Agency Partner > Click on all the buttons for individual directories > Verify the zip file gets downloaded > Unzip the file and verify that the files are correct. 

<img width="1412" alt="Screenshot 2024-10-23 at 1 51 37 PM" src="https://github.com/user-attachments/assets/6394783d-3c52-49dd-945e-bb1247a78a1f">

<img width="801" alt="Screenshot 2024-10-23 at 1 51 49 PM" src="https://github.com/user-attachments/assets/1abda132-8014-410b-bebf-22813c9c5b64">

* Change the agency tier to **Pro Agency Partner** > Refresh the UI and verify the screen looks as shown below. Click the `Download your badges` button > Verify all the directory badges are visible for Pro Agency Partner > Click on all the buttons for individual directories > Verify the zip file gets downloaded > Unzip the file and verify that the files are correct. 

<img width="811" alt="Screenshot 2024-10-23 at 1 23 06 PM" src="https://github.com/user-attachments/assets/e2a66101-534f-4b3c-b656-68e8380fee01">

* Verify it looks good on all the screen sizes:

![iPad-Mini-agencies localhost (1)](https://github.com/user-attachments/assets/554b2913-6367-4d7c-861e-d0f26bcbfea6)

![iPad-Mini-agencies localhost](https://github.com/user-attachments/assets/a5b0fe77-25e5-4cdc-bbb8-d906010c3e86)

![iPhone-12-PRO-agencies localhost (9)](https://github.com/user-attachments/assets/bec5a54d-5df1-4ddf-ac30-a6ee9852c911)

![iPhone-12-PRO-agencies localhost (10)](https://github.com/user-attachments/assets/ca2ba009-350a-4ee2-ae91-783e4248ddd1)



## Pre-merge Checklist
